database/dummyData.js

```diff
@@ -0,0 +1,93 @@
+// const Review = require('./db.js').Review;
+const moment = require('moment');
+const loremHipsum = require('lorem-hipsum');
+const dateGen = require('random-date-generator');
+const faker = require('faker');
+
+var genReviewText = () => {
+  return loremHipsum({
+    count: 3,
+    units: 'sentences',
+    sentenceLowerBound: 1,
+    sentenceUpperBound: 5
+  });
+};
+
+var genDate = () => {
+  dateGen.getRandomDate();
+  var startDate = new Date(2015, 1, 1);
+  var endDate = new Date(2019, 4, 5);
+  return dateGen.getRandomDateInRange(startDate, endDate);
+};
+
+var genReviewList = (amount) => {
+  var reviews = [];
+  var num = 0;
+  for (var i = 0; i < amount; i++) {
+    num = faker.random.number({min: 0, max: 99});
+    if (!reviews.includes(num)) {
+      reviews.push(num);
+    }
+  }
+  return reviews;
+};
+
+
+var createReview = () => {
+  var bulkData = [];
+  for (var k = 1; k <= 100; k++) {
+    var revAmount = faker.random.number({min: 7, max: 50});
+    var reviewArray = [];
+    var tempReview = {};
+    tempReview._id = k;
+    tempReview.accuracy = 0;
```

Comment:
Is it needed to define the properties on lines 43 - 49 if they are going to (essentially) be overwritten on lines 69 - 75?

Reply:
No, it is not, lines 43 - 49 is **tempSubReview** which is keeping values for each user review and calculate the total point and keep inside the tempReview.

After finishing the loop, line 69 - 75 keep the calculated values of average values inside **tempReview** again.